sources/api/shibaken/src/main.rs

```diff
@@ -0,0 +1,286 @@
+/*!
+# Introduction
+
+shibaken is called by sundog as a setting generator.
+
+shibaken will fetch and populate the host container's user-data with authorized ssh keys from the
```

Comment:
nit

```suggestion
shibaken will fetch and populate the admin container's user-data with authorized ssh keys from the
```

Reply:
@zmrow in several places you asked me you use host container's user-data previously. Should I go back and switch all of those to admin container's user-data? I'm kind of with Sam here in feeling that admin container is more correct.